src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -569,6 +611,17 @@ private bool TryGetUnmanagedCallingConventionFromModOpt(MethodSignature signatur
             return found;
         }
 
+        private static CorInfoCallConv? GetCallingConventionForCallConvType(DefType defType) =>
+            // Look for a recognized calling convention in metadata.
+            defType.Name switch
+            {
+                "CallConvCdecl" => CorInfoCallConv.CORINFO_CALLCONV_C,
```

Comment:
Do we want to use the `nameof()` pattern here?

Reply:
Crossgen2 doesn't use the `nameof()` pattern anywhere for referring to types in the target assembly.